### PR TITLE
Add autofill for wordpress installation - makes every flag optional.

### PR DESCRIPTION
**Pull request by niristius**
I prototyped a way to make every app-installation-flag optional.
if at least one flag is missing the missing data will be generated from the current mStudio Token or other fallback-data.
Looking for a better way to get the local OS-User as fallback in src/commands/app/install/wordpress.tsx:104 without the require statement
Also: Im trying to refine the processs with WordPress as an exemplary process.
Creating the Processes for further Apps will just be a bit of copy&paste.

### CODE REVIEW

File: src/lib/app/flags.tsx [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+    process.addInfo(
+      <Text>
+        Using generated random Admin Pass: <Value>{flags["admin-pass"]}</Value>
+      </Text>,
+    );
+  }
+
+  // Admin Firstname
+  if (necessaryFlags.includes("admin-firstname") && !flags["admin-firstname"]) {
+    flags["admin-firstname"] = ownUser.data.person!.firstName;
```
Comment (martin-helmich) [inline src/lib/app/flags.tsx:234]: We should test that `ownUser.data.person` is actually non-null.

File: src/lib/app/flags.tsx [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+      <Text>
+        Using mStudio firstname as Admin firstname (
+        <Value>{flags["admin-firstname"]}</Value>)
+      </Text>,
+    );
+  }
+
+  // Admin Lastname
+  if (necessaryFlags.includes("admin-lastname") && !flags["admin-lastname"]) {
+    flags["admin-lastname"] = ownUser.data.person!.lastName;
```
Comment (martin-helmich) [inline src/lib/app/flags.tsx:245]: Same here

File: src/lib/app/flags.tsx [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+    process.addInfo(
+      <Text>
+        Using mStudio firstname as Admin lastname (
```
Comment (martin-helmich) [inline src/lib/app/flags.tsx:248]:
```suggestion
        Using mStudio lastname as Admin lastname (
```

File: src/commands/app/install/contao.tsx
Comment (martin-helmich) [inline src/commands/app/install/contao.tsx:1]: Also, there's a new CI test that asserts that the `README.md` is up-to-date; can you run one final `yarn generate:readme`?

Comment (niristius) [reply src/commands/app/install/contao.tsx:1]: Ah, was searching for a way to generate it. Didn't think for it to be that easy👍🏼

**Review by martin-helmich — APPROVED** [review]

**Pull request merged**